frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -102,7 +110,9 @@ export function SelectEmbedTypePane({
         <SharingPaneButton
           title={t`Static embedding`}
           illustration={<StaticEmbeddingIllustration />}
-          onClick={goToNextStep}
+          onClick={isStaticEmbeddingDisabled ? undefined : goToNextStep}
```

Comment:
This is to avoid making the whole thing clickable when the setting is disabled. But rather than setting `pointer-events: none` since we need to be able to click the link that will lead us to enable the setting in the admin page at the bottom of the card.